pkg/crc/machine/start.go

```diff
@@ -398,8 +398,8 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 
 	waitForProxyPropagation(ocConfig, proxyConfig)
 
-	logging.Info("Updating kubeconfig")
-	if err := eventuallyWriteKubeconfig(ocConfig, instanceIP, clusterConfig); err != nil {
+	logging.Info("Adding crc-admin and crc-developer contexts to kubeconfig ...")
```

Comment:
As per @robin-owen comment on the issue https://github.com/code-ready/crc/issues/2039#issuecomment-788181671 , we should not have any space in between.

Reply:
thanks done!